scripts/installscripts/stretch-install-default.sh

```diff
@@ -296,70 +296,20 @@ clear
 
```

Comment:
We could delete the stretch file. AS far as I remember the installation on stretch doesn’t work anymore.